### PR TITLE
Init project with components

### PR DESCRIPTION
**Pull request by LOctavio**
On this milestone I did the following:

- Create a React and Redux app.
- Structure files using the "feature folder" approach.
- Set up routing using React Router.

### CODE REVIEW

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @LOctavio  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!


#  Required Changes ♻️ 🚧
Project is missing a **CreateNewBook** component

**CreateNewBook**
- Renders a form with inputs for the title and the author of the book
Has a button to submit
- the button wouldn't add the book yet, this is just a placeholder for now

- [ ] Kindly implement the **CreateNewBook** component

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const books = () => (
+  <form>
+    <input type="text" className="title" placeholder="title" />
+    <input type="text" className="author" placeholder="author" />
+    <button type="submit">submit</button>
+  </form>
```
Comment (oloomoses) [inline src/redux/books/books.js]: - [ ] The form should be rendered in the `CreateNewBook` component as suggested in the previous review. Kindly fix it.

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @LOctavio, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Slack: Oloo Moses
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/CreateNewBook.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const CreateNewBook = () => (
+  <form>
+    <input type="text" className="title" placeholder="title" />
+    <input type="text" className="author" placeholder="author" />
+    <button type="submit">submit</button>
+  </form>
+);
+
+export default CreateNewBook;
```
Comment (oxenprogrammer) [inline src/components/CreateNewBook.js:9]: **Nicely Done**
- [ ] Good effort working on the requested change by the previous Code Reviewer.

**Review by oxenprogrammer — APPROVED** [review]
Hi @LOctavio , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**